c2corg_ui/static/partials/contexthelp/route-equipement_rating.html

```diff
@@ -0,0 +1,28 @@
+<p><strong translate>Protection</strong></p>
```

Comment:
Perhaps consider using ``<h4 translate>`` instead. Ditto for other "strong" paragraphs below.